pkg/collector/corechecks/network/ntp.go

```diff
@@ -108,6 +115,11 @@ func (c *NTPCheck) Configure(data integration.Data, initConfig integration.Data)
 
 // Run runs the check
 func (c *NTPCheck) Run() error {
+	if time.Now().Before(c.lastCollection.Add(time.Second * time.Duration(c.cfg.instance.MinCollectionInterval))) {
```

Comment:
is `lastCollection` correctly set for the first run ?

Reply:
yes, as documented in the [go docs](https://golang.org/pkg/time/#Time): `The zero value of type Time is January 1, year 1, 00:00:00.000000000 UTC.`